### ISSUE TITLE
update results plot

### ISSUE THREAD

**Issue by rbcavanaugh** [rbcavanaugh/pnt#5]
Results plot needs a few updates. 

1. Question about whether it should be based on a normal distribution

- better to just show a scale (-4, 4) and plot the posterior distribution for the ability estimate instead
- this would allow multiple posterior distributions to be shown with their overlap

2. If we continue to show as is, distribution should reflect sample from study

**Comment by rbcavanaugh** [comment]
there is another, related issue here - how can we show an estimate for the distributions for each ability estimate? The fullDist function in catR doesn't seem to want to work. can we run a quick model using rstanarm? or should we just show a normal distribution where the distribution is Normal(ability_estimate, 1.96*SEM)?

**Comment by gfergadiotis** [comment]
I think Will should chime in, but I think that the ~N(θ, 1.96*SEM|θ) should work just fine.

**Comment by william-hula** [comment]
Yeah, if we're talking about the posterior distribution of an individual score estimate, I think the normal distribution is just fine. But if we're talking about plotting the current examinee's point estimate or posterior distribution in relation to the calibration sample, that's another matter, because as I commented in the other thread about scaling, the scores do not seem to be normally distributed in relatively large samples. What if you had an actual histogram of the score estimates from the calibration sample with the point estimate and 95%CI superimposed or printed beneath. You could display the CI as simple error bars, or, as i think you're suggesting, a normal density.

**Comment by gfergadiotis** [comment]
I agree with Will.

**Comment by rbcavanaugh** [comment]
Doable - send me a .csv with a vector of all the calibrated sample estimates

**Comment by rbcavanaugh** [comment]
uploaded.
